FlaskWeatherAPI/constants.py

```diff
@@ -0,0 +1,2 @@
+WEATHER_API_KEY = 'VJWG7USK9FMZNP9BB27NUADAK'
+WEATHER_API_TIMELINE_BASE_URL = 'https://weather.visualcrossing.com/VisualCrossingWebServices/rest/services/timeline'
```

Comment:
The typical way to approach this would be with a `.env` file. You can check the dotenv library as a starting point. As well as the built-in `os` library with methods such as `getenv()`